tests/lib/tools/fs-state

```diff
@@ -77,6 +81,60 @@ restore_file() {
     fi
 }
 
+save_current_tree(){
+    local dir=$1
+
+    rm -rf "$dir"
+    mkdir "$dir"
+    # .snap files created in tests/lib/snaps are excluded as they are not deleted to be reused
+    # runtime-state dir contains cursor and run log which are created during the first test
+    tree --noreport -if /etc -I "*~" > "$dir"/etc
+    tree --noreport -if /boot -I "*~" > "$dir"/boot
+    tree --noreport -if /home -I "*~" | grep -v -e "$PROJECT_PATH/tests/lib/snaps/.*/.*.snap" -e "$PWD/.*.snap" > "$dir"/home
+    tree --noreport -if /root -I "*~" > "$dir"/root
```

Comment:
well, yes, I started with tree because I was printing tree structure but then after face some errors I added -i and it became a find. I'll update this.